index.bs

```diff
@@ -3276,9 +3276,8 @@ socket.
 <h3 id="example-rbs-push">A readable byte stream with an underlying push source and backpressure support</h3>
 
 The following function returns <a>readable byte streams</a> that allow efficient direct reading of TCP sockets, based on
-a hypothetical JavaScript translation of the POSIX socket API (the main innovation of which is translating
-<code>select(2)</code> into an <code><a idl class="non-normative">EventTarget</a></code> emitting a
-<code>readable</code> event). [[DOM]]
+a hypothetical JavaScript socket API. The socket object is an <code><a idl class="non-normative">EventTarget</a></code> 
+which emits a <code>readable</code> event. [[DOM]]
```

Comment:
Removal of "translation of ..." is fine, but I think we should keep some text mentioning similarity with select(2). Inside parentheses is fine.